openc3/python/openc3/microservices/decom_microservice.py

```diff
@@ -179,15 +236,9 @@ def limits_change_callback(self, packet, item, old_limits_state, value, log_chan
         LimitsEventTopic.write(event, scope=self.scope)
 
         if item.limits.response is not None:
-            try:
-                # TODO: The limits response is user code and should be run as a separate thread / process
-                # If this code blocks it will delay TelemetryDecomTopic.write_packet
-                item.limits.response.call(packet, item, old_limits_state)
-            except Exception as error:
-                self.error = error
-                self.logger.error(f"{packet.target_name} {packet.packet_name} {item.name} Limits Response Exception!")
-                self.logger.error(f"Called with old_state = {old_limits_state}, new_state = {item.limits.state}")
-                self.logger.error(repr(error))
+            copied_packet = packet.deep_copy()
```

Comment:
Python already has `deepcopy`: https://docs.python.org/3/library/copy.html#copy.deepcopy. Should this just be `packet.deepcopy`?